api/SourceBuffer.json

```diff
@@ -1111,54 +1111,6 @@
           }
         }
       },
-      "trackDefaults": {
```

Comment:
I extracted this removal into https://github.com/mdn/browser-compat-data/pull/12703